addon/mixins/sortable-table.js

```diff
@@ -1,19 +1,21 @@
 import Mixin from '@ember/object/mixin';
 import { computed } from '@ember/object';
+
 export default Mixin.create({
   sortBy: '',
 
   sortedAscending: computed('sortBy', function(){
     const sortBy = this.get('sortBy');
     return sortBy.search(/desc/) === -1;
   }),
+
   actions: {
-    sortBy(what){
+    sortBy(prop) {
```

Comment:
Why make this change? It seems like unnecessary churn that adds to this diff. Should be a separate PR.